test/Microsoft.Extensions.Primitives.Tests/StringValuesTests.cs

```diff
@@ -294,7 +294,11 @@ public void Contains(StringValues stringValues, string[] expected)
 
         [Theory]
         [MemberData(nameof(FilledStringValuesWithExpected))]
-        public void CopyTo_TooSmall(StringValues stringValues, string[] expected)
+        public void CopyTo_TooSmall(
+            StringValues stringValues,
+#pragma warning disable xUnit1026 // Theory methods should use all of their parameters
+            string[] expected)
```

Comment:
Of course the shape of the MemberData needs to match the parameters, but I think it would be cleaner to remove the expected values from the member data, rather than supressing the xunit warning.  There is already a `FilledStringValues` TheoryData, but it has different keys than `FilledStringValuesWithExpected`.